### DISCUSSION TITLE
Configuration time benchmark

### DISCUSSION THREAD

**Discussion by esafirm** [slackhq/foundry#9] [Q&A] [answered]
Would be nice if slack posted some data about the configuration time that can be used as the baseline for other teams out there.

---

Our team has ~300 modules and it took about 1m 30s for configuration in CI and 14s in local with a warm daemon in MBP 15 2018.
We're not sure if this is fast or slow as the profiler result told us most of the works belong to the Android and Kotlin Gradle plugin.

**Comment by ZacSweers** [comment, marked as answer]
We have some benchmarks, I can try to share them here when I'm back from my trip

**Reply by esafirm** [reply]
Thanks Zac! You're the best!

**Reply by ZacSweers** [reply]
Our benchmarks have ~29 seconds on beefy CI AWS instance. We don't have any good local numbers for this (we don't run this benchmark on any local devices and most local builds have configuration cache enabled, so it's usually essentially negligible).

**Reply by esafirm** [reply]
Thanks! And this is for how many modules?

**Reply by ZacSweers** [reply]
~500
